tests/helper.js

```diff
@@ -36,6 +37,24 @@ before(function() {
   });
 });
 
-after(function() {
-  this.sequelize.close();
-})
+afterEach(async function() {
+  // https://github.com/sequelize/sequelize/blob/29901187d9560e7d51ae1f9b5f411cf0c5d8994a/test/support.js#L136
```

Comment:
Can we pull this logic into a helper function as well?

Reply:
Sure! Like this? 7a1bd15

Reply:
I prefer `cleanupDatabase` over `cleanDatabase`

Reply:
That's better indeed. 5e41170